sparkbox/apprentices-page-next
# Feat  spacing

## Description

- Refactored Hero component
- Added _spacing.scss and variables for consistent spacing on components
- Adjust tests for index

See Story: [FSA21V2-145](https://sparkbox.atlassian.net/browse/FSA21V2-145)

## To Test

1. Make sure all PR Checks have passed (Github Actions, Netlify etc).
2. Pull down all related branches.
3. Confirm all tests pass: `npm run test:ci`
4. run npm lint to make sure linter passes

## Validation

The following has been completed by the developer:


### Browser Testing

#### Gold Level Browsers

In these browsers, all content is accessible and design matches exactly. In most cases, bugs should be resolved before merge.

**macOS**


**Windows**


**Mobile**


#### Silver Level Browsers

In these browsers, all content is readable, but design may be inaccurate.

**Windows**


**Mobile**


#### Bronze Level Browsers

In these browsers, our only goal is content readability.

**Windows**
